client/starwhale/core/eval/executor.py

```diff
@@ -144,7 +147,10 @@ def _extract_swmp(self) -> None:
 
     def _extract_swrt(self) -> None:
         if self.runtime and self.use_docker:
-            self._runtime_dir = self.runtime.extract()
+            # avoid conflict with normal process with venv or conda
+            self._runtime_dir = self.runtime.extract(
```

Comment:
use `self.runtime.extract() ` directly, same as [swmp](https://github.com/star-whale/starwhale/pull/1539/files#diff-d0daabe4e624273df46ade2796e97724926a95a9ae307ba6d2fc21000248f0a1R146)?

Reply:
removed, now in one way